web/src/components/LocationLayout.tsx

```diff
@@ -73,7 +78,17 @@ const LocationLayout = (props: PropsType): ReactElement => {
       footer={!isLoading && showFooter ? <LocationFooter city={cityModel.code} language={languageCode} /> : null}
       modal={feedbackModal}
       toolbar={toolbar}>
-      {children}
+      <>
+        {children}
+        {buildConfig().featureFlags.developerFriendly && cityModel.name === 'München' && (
+          <div
+            id='rasa-chat-widget'
+            data-primary={theme.colors.themeColor}
+            data-primary-highlight={theme.colors.themeColor}
+            data-websocket-url='https://your-rasa-url-here/'
```

Comment:
seems not to be a working url? do we need to pass this?

Reply:
As i mentioned in the PR description, i do not know the correct url right now, but I asked and will update as soon as I get it.